ydb/core/tx/schemeshard/schemeshard_impl.cpp

```diff
@@ -4413,6 +4413,20 @@ TActorId TSchemeShard::TPipeClientFactory::CreateClient(const TActorContext& ctx
     return clientId;
 }
 
+TSchemeShard::TAccountLockout::TAccountLockout(const ::NKikimrProto::TAccountLockout& accountLockout)
+    : AttemptThreshold(accountLockout.GetAttemptThreshold())
+{
+    AttemptResetDuration = TDuration::Zero();
+    if (accountLockout.GetAttemptResetDuration().empty()) {
+        return;
+    }
+    if (TDuration::TryParse(accountLockout.GetAttemptResetDuration(), AttemptResetDuration)) {
+        if (AttemptResetDuration.Seconds() == 0) {
+            AttemptResetDuration = TDuration::Zero();
```

Comment:
Is it covered with unit tests?

Reply:
I think we cannot test this case. It is unclear how many login attempts need to be done

Reply:
May be something like `runtime->AdvanceCurrentTime(TDuration::Years(1));` ?